lib/app/endpoint.rb

```diff
@@ -51,6 +51,14 @@ class Endpoint < Sinatra::Base
       set :static, true
       set :views, File.expand_path('views', __dir__)
       set(:prefix) { '/' << name[/[^:]+$/].underscore }
+
+      def render_index
+        unless defined?(settings.presets).nil? || settings.presets.nil? # rubocop:disable Style/IfUnlessModifier
+          presets = settings.presets.select { |_, v| v['inferno_uri'].nil? || v['inferno_uri'] == request.base_url }
```

Comment:
I don't think this would match `https://inferno.healthit.gov/inferno`, as `request.base_url` is probably only `http://domain`, without path information.  One complication is that usually inferno can either be accessed at `http://domain/inferno` or `http://domain`, but that isn't the case on `healthit.gov` because the root is intended to show the community vs. program landing page.  And we have `http://domain/community` for the community instance.

I would recommend:
* look for exact match to the domain (base_url) here
* but also accept `v['inferno_url'] == request.base_url + Inferno::BASE_PATH`

Also, since we are dealing with user input in the config, perhaps we should get rid of any trailing slashes just in case, on both BASE_PATH and v['inferno_url'].  Otherwise people may put in `http://localhost/inferno/` but that doesn't match `http://localhost/inferno`, but logically they are the same.  Look into `chomp('/')`.

Reply:
Updated cases for matching